api/src/main/java/org/openmrs/PersonName.java

```diff
@@ -76,13 +78,15 @@
 			@Field(name = "familyNameExact", analyzer = @Analyzer(definition = LuceneAnalyzers.EXACT_ANALYZER), boost = @Boost(8f)),
 			@Field(name = "familyNameStart", analyzer = @Analyzer(definition = LuceneAnalyzers.START_ANALYZER), boost = @Boost(4f)),
 			@Field(name = "familyNameAnywhere", analyzer = @Analyzer(definition = LuceneAnalyzers.ANYWHERE_ANALYZER), boost = @Boost(2f)),
+			@Field(name = "familyNameSoundex", analyzer =  @Analyzer(definition = LuceneAnalyzers.SOUNDEX_ANALYZER), boost = @Boost(16f))
```

Comment:
Do we really want such a high boost for a SOUNDEX match? I would suggest the boost should be left at 1.0.

Reply:
I can set all to 1.0. It should have no effect I think 👍